src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -467,6 +467,21 @@ public Vector4 CornerRadius
         // Accessibility
         //
 
+        /// <summary>
+        /// Gets or sets a value that allows the automation framework to find and interact with this element.
+        /// </summary>
+        /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override string AutomationId
```

Comment:
How about change as followings
`public override string AutomationId` => `public new string AutomationId` 
`public virtual string AutomationId` => `public string AutomationId` (no change)
(personally, Element is base class and it looks not good to change base class's interface)